madgiraffestudios/src/components/Navigation/NavigationBar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
```

Comment:
- [ ] use reactstrap here - import should look something like below but google reactstrap to confirm:
import { Navbar, NavbarBrand, Nav, NavItem, NavLink } from 'reactstrap';